source/Junkership.js

```diff
@@ -3,43 +3,62 @@ var Keyb = require("keyb")
 
 import Reference from "./Reference.js"
 import Projectile from "./Projectile.js"
+import Score from "./Score.js"
 
 var junkerTex = Pixi.Texture.fromImage(require("./images/blue-starship.png"))
 
 export default class Junkership extends Pixi.Sprite {
     constructor() {
         super(junkerTex)
-        this.speed = 60
-        this.score = 0
         game.playerCount++
+        this.speed = 60
+        this.score = new Score()
     }
     update(delta) {
-        if(Keyb.isDown("<up>") && this.position.y > 0) {
-            this.position.y -= this.speed * delta
-        }
-        if(Keyb.isDown("<down>") &&
-            (this.position.y + this.height) < Reference.GAME_HEIGHT) {
-            this.position.y += this.speed * delta
+        var relativeSpeed = this.speed * delta
+        if(Keyb.isDown("<up>")) {
+            this.move(-relativeSpeed, "y")
+        } else if(Keyb.isDown("<down>")) {
+            this.move(relativeSpeed, "y")
```

Comment:
Mm, that sounds good. Let's move forward with this, but come back to it later ifneedbe.